backend/package.json

```diff
@@ -5,15 +5,17 @@
   "main": "build/server.js",
   "scripts": {
     "build": "rimraf build && tsc --build tsconfig.build.json",
-    "dev": "npm run postbuild && tsc-watch --onSuccess \"node ./build/server.js\"",
+    "dev": "npm run database:migrate && npm run database:seed && npm run postbuild && tsc-watch --onSuccess \"node ./build/server.js\"",
```

Comment:
Do we actually want to run seed on every dev server bootup though? Because the seeds might clash with existing prior seeds and cause error. (Not sure how your local setup is but I have a persistent volume for my local postgres docker and keep reusing it)

Reply:
Ah that's fair. Maybe we don't need this PR then.